src/Microsoft.AspNet.Mvc.Core/ModelBinding/FormCollectionModelBinder.cs

```diff
@@ -25,7 +25,7 @@ public class FormCollectionModelBinder : IModelBinder
                 return null;
             }
 
-            object model = null;
```

Comment:
Revert changes to this file. @rynowak doesn't like thinking of the whole `FormCollection` as user input.